switch_theme.cgi

```diff
@@ -0,0 +1,47 @@
+#!/usr/local/bin/perl
+# switch_theme.cgi
+# Change the theme for the current user if allowed
+
+BEGIN { push(@INC, "."); };
+use WebminCore;
+
+&init_config();
+&ReadParse();
+my $err = sub {
```

Comment:
Why not just call error() ?

Reply:
It won't look good in this case application.

Reply:
Why won't it look good?

Reply:
> Why won't it look good?

Because we're not in the module. Those extra buttons inside of a large container .. It's much cleaner just to print the message and exit.